docs/modules/ROOT/pages/how-to-guides/testing_applications/con_test-overview.adoc

```diff
@@ -35,4 +35,6 @@ Custom tests in {ProductName} are tests that users and administrators create. To
 
 Integration tests ensure that all build components are able to work together at the same time. You can xref:how-to-guides/testing_applications/proc_adding_an_integration_test.adoc[add an integration test], simply by giving {ProductName} the address to a GitHub repo, and the path within that repo to the `.yaml` file that defines the test.
 
-{ProductName} runs integration tests after it successfully builds the components of an application. As part of the build process, {ProductName} creates an image for each component and stores them in a Quay.io repository. Images of all the components are then compiled into a snapshot of the application. {ProductName} tests the snapshot against user-defined IntegrationTestScenarios, which, again, refer to a GitHub repository. 
\ No newline at end of file
+{ProductName} runs integration tests after it successfully builds the components of an application. As part of the build process, {ProductName} creates an image for each component and stores them in a Quay.io repository. Images of all the components are then compiled into a snapshot of the application. {ProductName} tests the snapshot against user-defined IntegrationTestScenarios, which, again, refer to a GitHub repository. 
+
+The integration service provides users with the ability to reset their component's Global Candidate List to a desired state with a manually created `override`. You can xref:how-to-guides/testing_applications/proc_creating_override_snapshot.adoc[create an override snapshot] and provide a valid container image for a component to reset the component's Global Candidate List.
```

Comment:
@hongweiliu17 sorry I meant to suggest a commit, not actually make one to your PR! Please feel free to revert from my changes. Although I just made a few slight changes for grammatical purposes.